Makefile

```diff
@@ -59,7 +59,7 @@ release: # create release
 .PHONY: test
 test:
 	mkdir -p $(COVERAGEDIR)
-	go test -short -v -coverprofile=$(COVERAGEDIR)/coverage.txt -covermode=atomic ./...
+	go test -gcflags=all=-d=variablemakehash=n -short -v -coverprofile=$(COVERAGEDIR)/coverage.txt -covermode=atomic ./...
```

Comment:
OK; looks like I may have found some clues; it's go1.25 exposing bugs on how unsafe.Pointers are used; https://go.dev/doc/go1.25#faster-slices

> Faster slices
>
> The compiler can now allocate the backing store for slices on the stack in more situations, which improves performance. This change has the potential to amplify the effects of incorrect [unsafe.Pointer](https://go.dev/pkg/unsafe#Pointer) usage, see for example [issue 73199](https://go.dev/issue/73199). In order to track down these problems, the [bisect tool](https://pkg.go.dev/golang.org/x/tools/cmd/bisect) can be used to find the allocation causing trouble using the `-compile=variablemake` flag. All such new stack allocations can also be turned off using `-gcflags=all=-d=variablemakehash=n`.
>